test/specs/getDirectoryContents.spec.js

```diff
@@ -35,6 +35,9 @@ describe("getDirectoryContents", function() {
             expect(details)
                 .to.have.property("headers")
                 .that.is.an("object");
+            expect(details.data[0])
```

Comment:
@perry-mitchell This makes on of the tests fail. I'm not familiar with this unit test framwork. What's the right way to do this?

Reply:
The only test I can see failing is a nodejs 6 test, but it's failing due to some pathing issues, which I believe were already fixed. Very strange. I'll try to investigate further shortly.

Reply:
I’ll most likely merge this tonight and will resolve any issues I find then.. thanks!